tools/cli/admin.go

```diff
@@ -615,3 +615,27 @@ func newAdminElasticSearchCommands() []cli.Command {
 		},
 	}
 }
+
+func newAdminTaskListCommands() []cli.Command {
+	return []cli.Command{
+		{
+			Name:    "describe",
+			Aliases: []string{"desc"},
+			Usage:   "Describe pollers and ack manager related info of tasklist",
```

Comment:
Its some what of an implementation detail that the contents of the stuff we are returning come from ack manager. A new cadence team member would not be super clear on what this is expected to return. What about something like "Describe pollers and task list status information"